GNUmakefile

```diff
@@ -64,6 +65,9 @@ format:
 	@echo "--> Running go fmt"
 	@go fmt $(PACKAGES)
 
+golint:
+	golint
```

Comment:
Did you check by running @golint $(PACKAGES)

Reply:
I didn't yet, I'll work on this changes tonight. I did test the way I have and it's only running on the root.